icicle/appUtils/ntt/kernel_ntt.cu

```diff
@@ -2,9 +2,9 @@
 #include "appUtils/ntt/thread_ntt.cu"
 #include "curves/curve_config.cuh"
 #include "utils/sharedmem.cuh"
-#include "appUtils/ntt/ntt.cuh" // for Ordering
+#include "appUtils/ntt/ntt.cuh" // for ntt::Ordering
 
-namespace ntt {
+namespace mxntt {
```

Comment:
why different namespace?

Reply:
otherwise  it's linking issue similar to the one when annonymous namespaces